### PR TITLE
fix(ui): Fix multi-app refresh and sync in the UI

### PR DESCRIPTION
**Pull request by jannfis**
I missed this in the [Applications in any namespaces](https://github.com/argoproj/argo-cd/pull/9755) PR.

Signed-off-by: jannfis <jann@mistrust.net>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Comment by ashutosh16** [comment]
@jannfis looks like this is missing the namespace, Does this need to fix as well?
  https://github.com/argoproj/argo-cd/blob/82746b6635ce11030cde0617d7b7ac3579a51a19/ui/src/app/applications/components/application-node-info/application-node-info.tsx#L123

**Comment by jannfis** [comment]
Good catch, @ashutosh16. I think there may be even more places throughout the UI's code where this is missing.

I'm trying to play catch 'em all, but unfortunately, the UI linter is not very helpful so I need to go full manual :\

Happy for suggestions.

File: ui/src/app/applications/components/application-node-info/application-node-info.tsx [hunk 1]
```diff
@@ -120,7 +120,9 @@ export const ApplicationNodeInfo = (props: {
                                 <YamlEditor
                                     input={live}
                                     hideModeButtons={!live}
-                                    onSave={(patch, patchType) => services.applications.patchResource(props.application.metadata.name, props.node, patch, patchType)}
+                                    onSave={(patch, patchType) =>
+                                        services.applications.patchResource(props.application.metadata.name, props.application.metadata.name, props.node, patch, patchType)
```
Comment (rbreeze) [inline ui/src/app/applications/components/application-node-info/application-node-info.tsx:124]: Should second argument to this fxn call be `props.application.metadata.namespace`?

**Review by rbreeze — CHANGES_REQUESTED** [review]
One small comment, otherwise LGTM

File: ui/src/app/applications/components/application-node-info/application-node-info.tsx (hunk 1 above)
Comment (jannfis) [reply ui/src/app/applications/components/application-node-info/application-node-info.tsx]: Good catch! Fixed.

**Review by rbreeze — APPROVED** [review]
LGTM!

**Pull request merged**